foca/security/auth.py

```diff
@@ -18,15 +18,16 @@
 
 
 def validate_token(token: str) -> Dict:
-    """
-    Validate JSON Web Token (JWT) Bearer token.
+    """Validate JSON Web Token (JWT) Bearer token.
 
-    Returns:
-        Token information.
+    Args:
+        token (str): JSON Web Token to be validated.
 
     Raises:
-        connexion.exceptions.Unauthorized: Raised if JWT could not be
-            successfully validated.
+        Unauthorized: Raised if JWT could not be successfully validated.
```

Comment:
`Unauthorized` is not a standard Python error, and I think it's important that we indicate what error is raised specifically.